src/widgets/code/CodeSandbox.spec.tsx

```diff
@@ -27,22 +27,14 @@ const mockMetaMixin = <T extends Constructor<WidgetBase<any>>>(Base: T, mockStub
 
 describe('CodeSandbox', () => {
 	it('renders', () => {
+		add('build-time-render', false, true);
 		const url = 'https://codesandbox.io/embed/dummy';
 
-		const h = harness(() => <CodeSandbox url={url} />);
-		h.expect(() => (
-			<div key="root" classes={[css.root]}>
-				<iframe classes={[css.root]} src={`${url}?autoresize=1&hidenavigation=1`} />
-			</div>
-		));
-	});
-
-	it('renders with build-time-render', () => {
 		const mockMeta = jest.fn().mockImplementation((input: any) => {
 			if (Intersection) {
-				return {
-					get: mockIntersectionGet
-				};
+					return {
```

Comment:
This indent looks pretty intense. Is it right?

Reply:
Nope. Hadn't run prettier yet.